src/ServiceControl.AcceptanceTests/Audit/When_an_error_message_fails_to_import.cs

```diff
@@ -0,0 +1,137 @@
+﻿namespace ServiceBus.Management.AcceptanceTests.Audit
+{
+    using System.Collections.Generic;
+    using System.Net;
+    using System.Threading.Tasks;
+    using EndpointTemplates;
+    using NServiceBus;
+    using NServiceBus.AcceptanceTesting;
+    using NServiceBus.AcceptanceTesting.Customization;
+    using NServiceBus.AcceptanceTests;
+    using NServiceBus.Faults;
+    using NServiceBus.Settings;
+    using NUnit.Framework;
+    using ServiceControl.Infrastructure;
+    using ServiceControl.Operations;
+
+    /// <summary>
+    /// This is a specific test https://github.com/Particular/PlatformDevelopment/issues/2464 - an error message that ends up in failed audits collection
+    /// To simulate this we used audit message that contains an artificial FailedQ header
+    /// </summary>
+    class When_an_error_message_fails_to_import : AcceptanceTest
+    {
+        [Test]
+        public async Task It_can_be_reimported()
+        {
+            //Make sure the audit import attempt fails
+            CustomConfiguration = config => { config.RegisterComponents(c => c.ConfigureComponent<FailOnceEnricher>(DependencyLifecycle.SingleInstance)); };
+
+            FailedAuditsCountReponse failedImportsCountReponse;
+
+            await Define<MyContext>()
+                .WithEndpoint<Sender>(b => b.When((bus, c) =>
+                {
+                    var ops = new SendOptions();
+                    ops.SetHeader(FaultsHeaderKeys.FailedQ, "SomeFailedQ");
+                    return bus.Send(new MyMessage(), ops);
+                }))
+                .WithEndpoint<Receiver>()
+                .Done(async c =>
+                {
+                    if (c.UniqueMessageId == null)
+                    {
+                        return false;
+                    }
+
+                    if (!c.WasReimported)
+                    {
+                        var countResult = await this.TryGet<FailedAuditsCountReponse>("/api/failedaudits/count");
+                        failedImportsCountReponse = countResult;
+                        if (countResult)
+                        {
+                            if (failedImportsCountReponse.Count > 0)
+                            {
+                                c.FailedImport = true;
+                                await this.Post<object>("/api/failedaudits/import", null, code =>
```

Comment:
I think this can be simplified to

```
if(await this.Post<object>("/api/failedaudits/import")) {
   c.WasReimported = true;
}
```

the Post method already checks for

`response.IsSuccessStatusCode`